.github/workflows/go-test.yml

```diff
@@ -17,20 +17,19 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Set up Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
         with:
           go-version-file: go.mod
-          cache: true
 
       - name: Verify code formatting
         run: |
-          go install mvdan.cc/gofumpt@v0.4
+          go install mvdan.cc/gofumpt@v0.5
           test -z $(gofumpt -l .)
 
       - name: Run linter
         if: always()
         run: |
-          go install honnef.co/go/tools/cmd/staticcheck@v0.4
+          go install honnef.co/go/tools/cmd/staticcheck@v0.4.3
```

Comment:
0.4 means 0.4.3 if 0.4.3 is newest 0.4. Please revert

Reply:
oops, right, didn't know